app/styles/components/influencer/profile-description/styles.less

```diff
@@ -1,36 +1,36 @@
 .profile-description {
+  display: flex;
+  font-size: 12px;
 
-  // Messing with positioning to show shadows around influencer image
-  margin-top: -10px !important;
-  padding-top: 10px;
-  padding-bottom: 10px;
-  padding-left: 3px;
+  .profile-picture {
+    padding-right: 10px;
 
-  img {
-    .img-shadow;
-    height: 134px;
-    width: 159px;
-  }
-
-  .media-heading {
-    margin-bottom: 10px;
-  }
 
-  .media-body {
-    h4, p.addr {
-      .text-ellipsis();
+    img {
+      .img-shadow;
+      width: 150px;
+      height: auto;
```

Comment:
You right on 10px by 1000px this will break the layer ;) but for some light diff this will be good. But yes I will force the width/height.